setup.sh

```diff
@@ -8,9 +8,16 @@
 # http://www.apache.org/licenses/LICENSE-2.0
 ##############################################################################
 
+# set -o xtrace
 set -o nounset
 set -o pipefail
 
+# Make sure VTX is enabled
```

Comment:
This validation needs to be optional and based on the CPU flags. BTW, this requires another PR.

Reply:
@dlux What about using the following command?

    if [ $(sudo virt-host-validate | grep "kvm is accessible" | awk -F':' '{ print $3 }') == 'PASS' ]; then
        echo "KVM is accessible"
    fi